build.zig

```diff
@@ -288,6 +298,7 @@ fn build_zkvm_targets(b: *Builder, main_exe: *Builder.Step, host_target: std.Bui
             .optimize = optimize,
             .root_source_file = b.path("pkgs/utils/src/lib.zig"),
         });
+        zeam_utils.addImport("datetime", pg.module("datetime"));
```

Comment:
is not having this giving an error, basically zkvm targets are what runs inside zkvm i.e. freestanding os, and for that you have not used datetime, so ideally zig shouldn't even be asking for it when its building